### DISCUSSION TITLE
Video Can't Be Played File Corrupt

### DISCUSSION THREAD

**Discussion by mikecreates86** [tsightler/ring-mqtt#779] [Support Q&A]
I am still trying unsuccesfully to get the Ring live streaming working. I entered all data and I have marginal success when using the _event URL even though it's not smooth. When I try to use the _live URL, and watch the camera in Firefox, I get a "Video can't be played because the file is corrupt" error. Doesn't work in Safari either.

In the log, I don't know if this is related but I'm seeing this:

Deactivating event stream due to signal from RTSP server (no more active clients or publisher ended stream)
2024-01-12T21:28:36.309Z ring-rtsp kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]

I'm running all of this on a Home Assistant Green and I have quadruple checked all my config things. The rest of the integration of Ring MQTT is working perfectly. PLEASE HELP!

Cameras are newer Floodlight Cam Pro and a Ring Video Doorbell Pro 2. None of them work for livestreaming.

**Comment by tsightler** [comment]
It's really quite difficult to help at all with a single line of logs, not really that useful.  More useful would be the full logs from an attempt to start a stream.  However, based on the fact that you have limited success even with the event stream, which quite literally does nothing but stream a file directly from AWS S3, with basically no changes, my gut would say that there is some networking related issue since both of the errors you describe would indicate that.

For example, "Video can't be played because the file is corrupt" usually indicates that so many RTP packets were dropped by the WebRTC connection that it couldn't even identify critical metadata needed for the stream.

On the other hand, the event stream just streams the MP4 file directly from S3 via HTTPS, so it will retry any dropped packets, however, if there are a lot of dropped packets, the file will not be able to be streamed fast enough to keep from stuttering, which sounds similar to what you describe.

I don't really know how I can help further with that, just about everything that could ever be said about troubleshooting streaming has already been written here, in the FAQ, discussions forums, issues, HA forums and Reddit, so I would mostly just be repeating myself.  In the end, every single time I've taken my personal time to troubleshoot this issue over the last 2 years with any individual user, it has come down to various network related issues, from poor/crowded 2.4 Ghz wifi, to APs dropping packets, to one time quite literally being a switch which seemed to not be able to handle the initial burst of RTP traffic and dropping a bunch of packets.

Unfortunately, as I have other significant demands on my time, and this project has grown from a few hundred users total, to >10,000, I just don't have time to help individuals troubleshoot their various issues.

If you can provide full logs from a streaming session, I'm willing to look through them, but I don't know that it will really show me anything.
